code/src/layouts/footer/components/footer-banner/FooterBanner.scss

```diff
@@ -0,0 +1,32 @@
+@import "@design-system";
+
+.footer-banner {
+  color: $white;
```

Comment:
Shouldn't we use `var(--spa-footer-text-color, $white)` or something like that?

Reply:
Yes, thank you for noticing, we should )